lib/ransack/adapters/active_record/base.rb

```diff
@@ -20,6 +22,22 @@ def ransacker(name, opts = {}, &block)
             .new(self, name, opts, &block)
         end
 
+        def ransacker_alias(alias_name, normal_name)
+          self._ransacker_aliases[alias_name.to_s] = normal_name.to_s
+        end
+
+        def params_with_ransacker_aliases(params)
+          params.keys.each do |k|
+            predicate = Predicate.detect_from_string(k)
+            attribute = k.sub(/_#{predicate}$/, Ransack::Constants::EMPTY)            
```

Comment:
Line is too long. [86/80]<br>Unnecessary spacing detected.<br>Trailing whitespace detected.